terraform/implementation/main.tf

```diff
@@ -320,19 +325,20 @@ spec:
 YAML
 }
 
-# Helm Releases
 
 resource "helm_release" "building_blocks" {
-  for_each      = local.services
-  repository    = "https://cdcgov.github.io/phdi-charts/"
-  name          = "phdi-playground-${terraform.workspace}-${each.key}"
-  chart         = "${each.key}-chart"
-  recreate_pods = true
-  depends_on    = [helm_release.agic]
+  for_each        = var.services_to_chart
+  repository      = "https://cdcgov.github.io/phdi-charts/"
+  name            = "phdi-playground-${terraform.workspace}-${each.key}"
+  chart           = each.value
+  depends_on      = [helm_release.agic]
+  force_update    = true
+  recreate_pods   = true
+  cleanup_on_fail = true
 
   set {
     name  = "image.tag"
-    value = "latest"
+    value = "v1.1.1"
```

Comment:
The `latest` version of each Docker image wasn't being picked up by Terraform, so each deployment was at a wildly different image version (whatever had been `latest` when the service was first deployed). The workflow to make changes is therefore:
* Make your changes to the Docker container
* Publish changes as new release version
* Update this image tag
* Push these changes and re-deploy to desired env